test/runtests.jl

```diff
@@ -0,0 +1,77 @@
+using LambdaMaker
+using Test
+
+SAMPLE_FILENAME = "Dockerfile"
+
+@testset "_copy_file" begin
+    RESOURCE_DIR = mkdir("resources")
+
+    try
+        file_path = joinpath(RESOURCE_DIR, SAMPLE_FILENAME)
+
+        @testset "dest file dne" begin
```

Comment:
```suggestion
        @testset "dest file does not exist" begin
```
First time I see this acronym, maybe I'm too old school 😄 
Better to be explicit?

Reply:
I think it's pretty common, but never a bad idea to be explicit!